### PR TITLE
Adding 200GB to Empty VM and adding ubuntu 22.04 cloud image

### PR DESCRIPTION
**Pull request by saviodsouza**
Before submitting a pull request, please check the following.

---
When updating an **existing** appliance:
- [x] The new version is on top.
- [x] The filenames in the "images" section are unique, to avoid appliances / version overwriting each other.
- [x] If you forked the repo, running check.py doesn't drop any errors for the updated file.
---
When creating a **new** appliance:
- It's tested locally, i.e.
  - [x] You dragged an instance into a project on your box, got it installed (if necessary), and did some basic network checks (ping, UI reachable, etc.).
  - [x] GNS3 VM can run it without any tweaks.
  - [x] The device is in the right category: router, switch, guest (hosts), firewall
  - [x] You filled in as much info as possible (checks the schemas and other appliance files for some guidance).
- [x] When adding a container: it builds on Docker Hub and can be pulled.
- [x] The filenames in the "images" section are unique (to avoid appliances and/or versions overwriting each other).
- [x] If you forked the repo, running check.py doesn't drop any errors for the new file.
- [x] *Optional: a symbol has been created for the new appliance.*

### CODE REVIEW

**Review by grossmj — COMMENTED** [review]
We already have an empty VM applicance there: https://github.com/GNS3/gns3-registry/blob/master/appliances/empty-vm.gns3a

This is the one that should be changed.

Thanks 👍

**Comment by saviodsouza** [comment]
Yes🙂

Thank you

**Review by saviodsouza — COMMENTED** [review]
I hope these changes are ok

**Comment by grossmj** [comment]
There are still 2 empty VM files: `Empty-VM.gns3a` & `empty-vm.gns3a` I think we don't need `Empty-VM.gns3a` :)

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ __pycache__
 build
 packer_cache
 output-qemu
+.history/
```
Comment (grossmj) [inline .gitignore:8]: I think it can just be .history (no need for the /)

File: appliances/empty-vm.gns3a [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
     "category": "guest",
-    "description": "An empty VM with empty hard disks 8G, 30G & 100G.",
+    "description": "A empty VM with empty hard disks 8G, 30G, 100G & 200G.",
     "vendor_name": "GNS3",
     "vendor_url": "https://gns3.com",
     "documentation_url": "",
     "product_name": "QEMU",
     "product_url": "https://sourceforge.net/projects/gns-3/files/Empty%20Qemu%20disk/",
-    "registry_version": 4,
+    "registry_version": 5,
```
Comment (grossmj) [inline appliances/empty-vm.gns3a:11]: Is there a reason to go for registry_version 5 ?

**Review by grossmj — CHANGES_REQUESTED** [review]

File: appliances/empty-vm.gns3a (hunk 2 above)
Comment (saviodsouza) [reply appliances/empty-vm.gns3a:11]: I thought it was a increment to each update

**Comment by saviodsouza** [comment]
> There are still 2 empty VM files: `Empty-VM.gns3a` & `empty-vm.gns3a` I think we don't need `Empty-VM.gns3a` :)

I am currently working on a windows i can only see Empty-VM.gns3a in the explorer 😐

Can you fix this from your side please....

**Comment by grossmj** [comment]
No worries, I will fix this on my side :)

**Review by grossmj — APPROVED** [review]

**Comment by grossmj** [comment]
Thanks a lot 👍

**Pull request merged**